wsdl-core/src/main/java/io/ballerina/wsdl/core/WsdlOperation.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.wsdl.core;
+
+public class WsdlOperation {
```

Comment:
IMO this should be a Java record. And let's add some doc-comments. 

AFAIK this doc-comment related issues are captured with checkstyles plugin. Please make sure that we have configured the checkstyles plugin is configured correctly in the build.